### PR TITLE
Fixing navigation

### LINKED ISSUE #177: Navigation its incorrect
When the user select a profile of an employee from the ranking list and tap on the back button the user returns to their own profile view.

Expected:
The user should return to the ranking list
Actual:
The user return to their own profile

### PR DESCRIPTION
**Pull request by EduardoChuks**
Fix #177 
navigation from account profile with navigate back fixed

### CODE REVIEW

File: app/src/main/java/com/belatrixsf/connect/ui/contacts/ContactsListActivity.java [hunk 1]
```diff
@@ -44,4 +47,13 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
...
     }
 
+    public static Intent makeIntent(Context context) {
+        return new Intent(context, ContactsListActivity.class);
+    }
+
+    @Override
+    public void onBackPressed() {
+        super.onBackPressed();
+        Constants.PARENT_ACTIVITY_INDEX = Constants.PARENT_USER;
```
Comment (icerrate) [inline app/src/main/java/com/belatrixsf/connect/ui/contacts/ContactsListActivity.java]: D:

File: app/src/main/java/com/belatrixsf/connect/ui/home/UserActivity.java [hunk 2]
```diff
@@ -147,6 +149,7 @@ public boolean onNavigationItemSelected(MenuItem item) {
                     case R.id.menu_contacts:
                         Intent intent = new Intent(UserActivity.this, ContactsListActivity.class);
                         startActivity(intent);
+                        Constants.PARENT_ACTIVITY_INDEX = Constants.PARENT_CONTACTS;
```
Comment (icerrate) [inline app/src/main/java/com/belatrixsf/connect/ui/home/UserActivity.java]: D:

File: app/src/main/java/com/belatrixsf/connect/ui/contacts/ContactsListActivity.java (hunk 1 above)
Comment (PedroCarrillo) [inline app/src/main/java/com/belatrixsf/connect/ui/contacts/ContactsListActivity.java]: khe ez ezt0h

File: app/src/main/java/com/belatrixsf/connect/ui/home/UserActivity.java (hunk 2 above)
Comment (PedroCarrillo) [inline app/src/main/java/com/belatrixsf/connect/ui/home/UserActivity.java]: noooo pls stop

File: app/src/main/java/com/belatrixsf/connect/ui/contacts/ContactsListActivity.java (hunk 1 above)
Comment (PedroCarrillo) [inline app/src/main/java/com/belatrixsf/connect/ui/contacts/ContactsListActivity.java]: You are creating an static variable in constants which you are modifying it on a back pressed? Can you describe the problem? at least a not so bad approach would be check the instance of the activity to ContactList but still that's a bad approach.

**Pull request merged**